src/content/docs/en/reference/dev-toolbar-app-reference.mdx

```diff
@@ -171,6 +171,20 @@ This event can be sent to inform the user that the app requires attention.
 
 Indicates whether or not the app has a notification for the user. When `true`, the app icon will be highlighted using a red dot. Conversely, when `false`, the highlight will be removed. If this property is not specified, `true` will be assumed.
 
+#### `level: 'info' | 'warning' | 'error'`
+
+Indicates the level of the notification. This will be used to determine the color of the highlight on the app icon. If this property is not specified, `'error'` will be assumed.
```

Comment:
```suggestion
#### `level: ''error' | 'warning' | 'info'`

Indicates the level of the notification. This will be used to determine the color and shape (dark pink circle, gold triangle, or blue square) of the highlight on the app icon. If this property is not specified, `'error'` will be assumed.
```